packages/helm-charts/celo-fullnode/rc1-blockscout-archive-nodes.yaml

```diff
@@ -0,0 +1,148 @@
+aws: false
+azure: false
+fullnameOverride: false
+extraPodLabels:
+  stack: blockscout
+  mode: archive
+gcp: true
+genesis:
+  epoch_size: 17280
+  genesisFileBase64: ""
+  network: rc1
+  networkId: 42220
+  useGenesisFileBase64: false
+geth:
```

Comment:
does it make sense to add default values to the values.yaml file, where they are missing?

Reply:
Yes, I plan to organize the values file with https://github.com/celo-org/infrastructure/issues/409